src/scenarios/shared/precommands.py

```diff
@@ -276,8 +276,7 @@ def _build(self, configuration: str, framework: str = None, output: str = None,
                            [framework],
                            output is None,
                            None,
-                           *['--output', output] if output else [],
-                           *build_args)
+                           (['--output', output] if output else []) + build_args)
```

Comment:
Just curious, how was this failing before?

Reply:
I suspect the problem is that `*['--output', output] if output else []` is the same as `(*['--output', output]) if output else []`, The array unpacking is only applying to the first case and not the 2nd, so if `output` is empty then instead of passing in no parameters, it passes in an empty array

Reply:
looks like this commit changed the method to take a list rather than a star-args list of args https://github.com/dotnet/performance/pull/2629/files#diff-bb114884f249cbe31f5bc1d08f1fa3ab45f9229180b94880a0f0cc76467337ee

Reply:
Yes, the change from star-args to plain list.